### PR TITLE
Site equipment page

### PR DESCRIPTION
**Pull request by LukeDonmoyer**

### CODE REVIEW

**Comment by LukeDonmoyer** [comment]
Don't merge until after relative paths.

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/* Error: Can't find stylesheet to import.
```
Comment (ShaunJorstad) [inline src/App.css]: What is the purpose of this file?

File: src/assets/App.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/* Error: Can't find stylesheet to import.
```
Comment (ShaunJorstad) [inline src/assets/App.css]: What is the purpose of this file?

**Comment by LukeDonmoyer** [comment]
Ready for review.

**Review by ShaunJorstad — APPROVED** [review]

**Pull request merged**